api/openapi-spec/swagger.json

```diff
@@ -11857,6 +11857,552 @@
       },
       "type": "object"
     },
+    "io.k8s.api.flowcontrol.v1alpha1.FlowDistinguisherMethod": {
```

Comment:
this looks spurious... regenerate?

Reply:
Uhm... still getting it. I had to manually delete the change

Reply:
rebase on `master`, `make clean`, maybe?

Reply:
it worked the 2nd time 🤷‍♂️